test/contracts/common/erc20-transfer.ts

```diff
@@ -0,0 +1,79 @@
+import {ExpectTokenTransfer} from '../bond/single-collateral-bond-events'
+import {ActualTokenTransfer} from '../bond/verify-single-collateral-bond-events'
+import {BigNumber, Event} from 'ethers'
+import {TransferEvent} from '../../../typechain-types/IERC20'
+import {expect} from 'chai'
+import {Result} from '@ethersproject/abi'
+
+export function deepEqualsERC20TokenTransfer(
+    actual: ActualTokenTransfer,
+    expected: ExpectTokenTransfer
+): boolean {
+    return (
+        actual.to === expected.to &&
+        actual.from === expected.from &&
+        actual.value.toBigInt() === expected.amount
+    )
+}
+
+/**
+ * Shape check and conversion for a TransferEvents.
+ */
+export function erc20TransferEvents(events: Event[]): {
+    from: string
+    to: string
+    value: BigNumber
+}[] {
+    const converted: {
+        from: string
+        to: string
+        value: BigNumber
+    }[] = []
+
+    for (let i = 0; i < events.length; i++) {
+        const transfer = events[i] as TransferEvent
+        expect(transfer.args).is.not.undefined
+
+        const args = events[i].args
+        expect(args?.from).is.not.undefined
+        expect(args?.to).is.not.undefined
+        expect(args?.value).is.not.undefined
+
+        converted.push(transfer.args)
+    }
+
+    return converted
+}
+export type ExpectedERC20Transfer = {
+    to: string
+    from: string
+    amount: bigint
+}
+
+export function erc20TransferEventLogsFromResult(
```

Comment:
suggestion: name to `erc20TransferLogEvents` 

Consistency with the other method transfer event method and other classes doing the same thing.

If you're super keen on refactoring all the naming, then a dedicate PR is the way to go 🙂

Reply:
done!